docs/reference/bpmn-processes/error-events/error-events.md

```diff
@@ -3,49 +3,64 @@ id: error-events
 title: "Error events"
 ---
 
-Error events are events which reference an error. They are used to handle business errors in a process.
+Error events are events which reference an error.
+They are used to handle business errors in a process.
 
 ![process](assets/error-events.png)
 
 An error indicates that some kind of business error has occurred which should be handled in the process, for example, by taking a different path to compensate the error.
 
 ## Defining the error
 
-An error can be referenced by one or more error events. It must define the `errorCode` (e.g. `Invalid Credit Card`) of the error.
+An error can be referenced by one or more error events.
+It must define the `errorCode` (e.g. `Invalid Credit Card`) of the error.
 
 The `errorCode` is a `string` that must match to the error code that is sent by the client command or from the error end event.
 
-## Catching the error
+## Throwing the error
 
-An error can be caught using an error **boundary event** or an error **event subprocess**.
+You can throw an error from a **client command** while processing a job.
```

Comment:
this section might be easier to read if triggers and effects are split:

The error can be thrown while processing a job. Or it can be thrown inside a process using the end event.

Image

Errors thrown in job disable the job and ...
Errors thrown in end events do not stop the execution of other branches in the same process. However, the error event "bubbles up" through the scopes until it is caught.

Which also gives a transition to the next section

Reply:
This was difficult, so I took another route. WDYT about it now?